packages/angular_devkit/schematics_cli/bin/schematics.ts

```diff
@@ -267,7 +267,8 @@ export async function main({
       return argv._;
     }
   });
-  delete parsedArgs._;
+
+  parsedArgs._  = [];
```

Comment:
The behavior is now different.  Does this have any effect on usages of the property?

Reply:
This property is marked as non optional thus it cannot be deleted.

From the schematics it seems that an undefined or empty array have the same behaviour.